setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-from nose_launchable.version import __version__
+#from nose_launchable.version import __version__
```

Comment:
I'll fix it after code reviews.

BTW,  `pip install -e .` failed by this, is this intentional...?

Reply:
Hmm, no, I personally use `python setup.py install` to install it locally so I've never noticed that...

https://github.com/launchableinc/nose-launchable#install-nose-launchable-locally